raster/src/main/scala/geotrellis/raster/ConstantTile.scala

```diff
@@ -243,6 +249,46 @@ object ConstantTile {
       case ct: FloatCells => FloatConstantTile.fromBytes(bytes, cols, rows, ct)
       case ct: DoubleCells => DoubleConstantTile.fromBytes(bytes, cols, rows, ct)
     }
+
+  /**
+   * Create a [[ConstantTile]] that is set to the nodata value of the given celltype.
+   *
+   * @param   t      The [[CellType]] of the new [[ConstantTile]], it should have a NoData value.
+   * @param   cols   The number of columns that the new [[ConstantTile]] should have
+   * @param   rows   The number of rows that the new [[ConstantTile]] should have
+   * @return         The new [[ConstantTile]]
+   */
+  def emptyTile(t: CellType, cols: Int, rows: Int): Tile = {
+    if(t.isInstanceOf[NoNoData]) {
+      throw new IllegalArgumentException(s"Can not construct an empty constant tile for a celltype without nodata handling: $t")
```

Comment:
Why is it not possible?

Reply:
Not sure, but what would I use as nodata value?
Perhaps for Float and Double celltype, NaN is fine, but what about IntCellType? Just use 0 in that case?

Reply:
Default to the CellType NoData i.e. for int its the int nodaa, for float its float nodata and for double it is the double nodata (min values)

Reply:
https://github.com/locationtech/geotrellis/blob/24f62711f6886eb64bc1622563d6213d5ee0915e/raster/src/main/scala/geotrellis/raster/package.scala#L105-L111

Reply:
I'm looking through the codebase and I think this method should be called `empty` and implementation is probably a +- copy paste of https://github.com/locationtech/geotrellis/blob/master/raster/src/main/scala/geotrellis/raster/ArrayTile.scala#L451-L461

Reply:
rename done, this new method is indeed very similar to the one in ArrayTile, guess it's not bad to complete the API like this.
Also removed the exception, added more cases.